tests/sentry/snuba/metrics/test_metrics_query_layer/test_metrics_query_layer.py

```diff
@@ -0,0 +1,47 @@
+"""
+Metrics Service Layer Tests for Performance
+"""
+
+import pytest
+from freezegun import freeze_time
+from snuba_sdk.metrics_query import MetricScope, MetricsQuery
+from snuba_sdk.timeseries import Metric, Timeseries
+
+from sentry.sentry_metrics import indexer
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+from sentry.snuba.metrics.naming_layer import TransactionMRI
+from sentry.snuba.metrics_layer.query import resolve_metrics_query
+from sentry.testutils.cases import BaseMetricsLayerTestCase, TestCase
+
+pytestmark = pytest.mark.sentry_metrics
+
+
+@freeze_time(BaseMetricsLayerTestCase.MOCK_DATETIME)
+class MetricsQueryLayerTest(BaseMetricsLayerTestCase, TestCase):
+    @property
+    def now(self):
+        return BaseMetricsLayerTestCase.MOCK_DATETIME
+
+    def test_resolve_metrics_query(self):
+        self.store_performance_metric(
+            name=TransactionMRI.DURATION.value,
+            project_id=self.project.id,
+            tags={},
+            value=1,
+        )
+        metrics_query = MetricsQuery(
+            query=Timeseries(Metric(mri=TransactionMRI.DURATION.value), aggregate="count"),
+            scope=MetricScope(
+                org_ids=[self.project.organization_id],
+                project_ids=[self.project.id],
+                use_case_id="transactions",
```

Comment:
Should probably use the enum in case someone refers to tests as source of truth for how to use `MetricScope`
```suggestion
                use_case_id=UseCaseID.TRANSACTIONS.value,
```